include/tvm/script/printer/printer.h

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+#ifndef TVM_SCRIPT_PRINTER_PRINTER_H_
+#define TVM_SCRIPT_PRINTER_PRINTER_H_
+
+#include <tvm/node/node.h>
+#include <tvm/script/printer/ir_docsifier.h>
+
+#include <unordered_map>
+#include <vector>
+
+namespace tvm {
+namespace script {
+namespace printer {
+
+/*! \brief Default values in the TVMScript printer */
+struct Default {
+  /*! \brief Default data type of TIR buffer */
+  DataType buffer_dtype = DataType::Float(32);
+  /*! \brief Default data type of integer literals */
+  DataType int_dtype = DataType::Int(32);
+  /*! \brief Default data type of float literals */
+  DataType float_dtype = DataType::Void();
```

Comment:
Why is it `DataType::Void()`?

Reply:
I looked up the current convention and it seems that we always print T.float32/float64 for floating point numbers, so this is to be consistent with the current style

Reply:
Please remove it if it is not used. All at least add a comment for better readability.

Reply:
I do think that we could consider a default dtype for float in the near future

Reply:
I think we can change the comment to "Right now we always print out the explicit type of floating point values, so setting it to Void means we do not print without T.float32/T.float64 wrapper"

Reply:
that sounds good!